tensorflow/core/kernels/mkl/mkl_einsum_op.cc

```diff
@@ -115,14 +115,57 @@ struct MklEinsumHelper {
         MklMatMulPrimitiveFactory<T, T, T, T>::Get(
             *params, false /* value for do_not_cache */);
 
+    T* weight_data = const_cast<T*>(rhs.flat<T>().data());
+
+#ifdef DNNL_AARCH64_USE_ACL
+    memory::format_tag weight_format;
+    switch (params->b_dims.size()) {
+      case 2:
+        weight_format =
+            trans_y ? memory::format_tag::ba : memory::format_tag::ab;
```

Comment:
I think transpose situations are already handled by Line 110 for up to 12D cases, not only up to 5D.